plugins/inputs/mqtt_consumer/mqtt_consumer.go

```diff
@@ -184,6 +184,7 @@ func (m *MQTTConsumer) Init() error {
 	}
 
 	m.opts = opts
+	m.messages = make(map[telegraf.TrackingID]bool)
```

Comment:
not a big deal, but I prefer not using make when there's no allocation size
```suggestion
	m.messages = map[telegraf.TrackingID]bool{}
```